Configuration/PyReleaseValidation/python/relval_steps.py

```diff
@@ -1235,14 +1235,14 @@ def lhegensim(fragment,howMuch):
 
 # step4
 step4Defaults = { 
-                  '-s'            : 'ALCA:TkAlMuonIsolated+TkAlMinBias+MuAlOverlaps+EcalESAlign',
+                  '-s'            : 'ALCA:TkAlMuonIsolated+TkAlMinBias+MuAlOverlaps+EcalESAlign+EcalTrg',
                   '-n'            : 1000,
```

Comment:
@shervin86 apologies for the delay. New Global Tags are available now.
In any case this branch doesn't merge anymore.  
Please cherry-pick the commit at https://github.com/cms-sw/cmssw/pull/1492